packages/suite/src/components/wallet/PrivacyAccount/AnonymityLevelSlider.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { useCallback, useState, ChangeEventHandler } from 'react';
+import styled from 'styled-components';
+import { Range, P, variables } from '@trezor/components';
+import { useSelector, useActions } from '@suite-hooks';
+import { selectSelectedAccount } from '@wallet-reducers/selectedAccountReducer';
+import { selectCurrentTargetAnonymity } from '@wallet-reducers/coinjoinReducer';
+import * as coinjoinActions from '@wallet-actions/coinjoinAccountActions';
+
+const Container = styled.div`
+    display: flex;
+    flex-direction: column;
+`;
+
+const Slider = styled(Range)`
+    padding-bottom: 50px;
+    padding-top: 60px;
+    margin-top: 0;
+
+    cursor: pointer;
+    background: none;
+`;
+
+const LabelsWrapper = styled.div`
+    display: flex;
+    justify-content: space-between;
+    pointer-events: none;
+
+    margin-top: -46px;
+    margin-bottom: -16px;
+    margin-left: 10px;
+`;
+
+const Label = styled(P)<{ $offset?: number }>`
+    position: relative;
+    left: ${({ $offset }) => $offset ?? 0}px;
+    font-size: ${variables.FONT_SIZE.TINY};
+    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+    opacity: 0.5;
+`;
+
+interface AnonymityLevelSliderProps {
+    className?: string;
+}
+
+const trackStyle = {
+    background:
+        'linear-gradient(270deg, #2A9649 0%, #95CDA5 45%, #C8B883 55%, #C8B882 60%, #BF6767 100%);',
+};
+
+const minPosition = 0;
+const maxPosition = 100;
+
+const minValue = Math.log(1);
+const maxValue = Math.log(100);
+
+const scale = (maxValue - minValue) / (maxPosition - minPosition);
+
+const getValue = (position: number) =>
+    Math.round(Math.exp((position - minPosition) * scale + minValue));
+const getPosition = (value: number) => minPosition + (Math.log(value) - minValue) / scale;
+
+export const AnonymityLevelSlider = ({ className }: AnonymityLevelSliderProps) => {
+    const currentAccount = useSelector(selectSelectedAccount);
+    const targetAnonymity = useSelector(selectCurrentTargetAnonymity);
+
+    const { coinjoinAccountUpdateAnonymity } = useActions({
```

Comment:
We are using only `useDispatch` in mobile because it's kind of standart a we like it more. For web app use whatever you like more, both approaches seems equal. `useActions` is somehow memoized but it have zero effect because nearly every usage that I saw was then wrapped in some kind of anonymous functions later in callback and that counter all benefits of memo or there is another unrelated things that will cause component rerender anyway.

Also I really like that every action is wrapped in `dispatch()` so I iminitiately know that it's action/thunk, but that's matter of personal taste too.

Reply:
I like it for this reason too. I also like that you don't need to have a chunky `useActions()` if you have several actions., or **rename** imported actions or mport them **as `* as`**

Reply:
I don't wanna do it as part of this pull request.